File: app/src/main/java/com/karim/booksapp/data/models/pojos/BooksEntry.java

package com.karim.booksapp.data.models.pojos;

import java.util.List;
import com.squareup.moshi.Json;

public class BooksEntry {

    @Json(name = "kind")
    public String kind;
    @Json(name = "totalItems")
    public Integer totalItems;
    @Json(name = "items")
    public List<Item> items = null;

}

File: app/src/main/java/com/karim/booksapp/data/models/pojos/BooksEntry.kt
package com.karim.booksapp.data.models.pojos

import com.squareup.moshi.Json

class BooksEntry {

    @Json(name = "kind")
    var kind: String? = null
    @Json(name = "totalItems")
    var totalItems: Int? = null
    @Json(name = "items")
    var items: List<Item>? = null

}
